### PR TITLE
docs: update vfolder images and descriptions

### PR DESCRIPTION
**Pull request by ironAiken2**
### This PR is about updating vfolder section for 24.09

**Changes:**
- Added a feature description for the new folder creation modal and updated related images.

**Related PR:**
- https://github.com/lablup/backend.ai-webui/pull/2745 @ironAiken2 
- https://github.com/lablup/backend.ai-webui/pull/2529 @ironAiken2 

PDF[ko] : [데이터 & 폴더 활용하기 — Backend.AI Web-UI User Guide 24.03 문서.pdf](https://github.com/user-attachments/files/17488475/Backend.AI.Web-UI.User.Guide.24.03.pdf)

PDF[en] : [Handling Data & Storage Folders — Backend.AI Web-UI User Guide 24.03 documentation.pdf](https://github.com/user-attachments/files/17488477/Handling.Data.Storage.Folders.Backend.AI.Web-UI.User.Guide.24.03.documentation.pdf)

### CODE REVIEW

**Comment by ironAiken2** [comment]
* **#89** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-docs-webui/89?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ironAiken2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 1]
```diff
@@ -468,15 +477,17 @@ msgstr ""
...
-#: ../../vfolder/vfolder.rst:237
+#: ../../vfolder/vfolder.rst:240
 msgid ""
 "Click one of the model cards and you will see the modal that describes "
 "what is all about the model. If you want to clone the current model card "
 "to your folder, click 'Clone to a folder' button on the right top of the "
 "modal."
 msgstr ""
+"모델 카드를 클릭하면 모델에 대한 설명이 포함된 모달이 표시됩니다. "
+"선택된 모델을 폴더로 복제하려면 모달의 오른쪽 상단에 있는 '폴더로 복제'버튼을 클릭합니다. "
```
Comment (YEONFEEL96) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:488]:
```suggestion
"모달의 오른쪽 상단에 있는 '폴더로 복제' 버튼을 클릭하면 선택된 모델을 폴더로 복제할 수 있습니다. "
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 2]
```diff
@@ -468,15 +477,17 @@ msgstr ""
 "사용해봅시다. 모델 카드 중 하나를 클릭하면, 모델에 대한 모든 정보를 나타내는 모달이 띄워지게 됩니다. 현재 모델 카드를 내 "
```
Comment (YEONFEEL96) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:477]:
```suggestion
"사용해봅시다. 모델 카드 중 하나를 클릭하면, 모델에 대한 모든 정보를 나타내는 모달이 나타납니다. "
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 3]
```diff
@@ -468,15 +477,17 @@ msgstr ""
...
 "폴더로 가져오고 싶다면, 오른쪽 위의 'Clone to a folder' 버튼을 클릭해주세요."
```
Comment (YEONFEEL96) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:478]:
```suggestion
"오른쪽 위의 'Clone to a folder' 버튼을 클릭하면 현대 모델 카드를 나의 폴더로 가져올 수 있습니다. "
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 4]
```diff
@@ -413,17 +422,18 @@ msgstr ""
 "있습니다. 바로 모델 서비스를 시작하려면 제어 열의 '모델 서빙' 버튼을 클릭합니다. 'model_folder' 라는 폴더에 대해 "
```
Comment (YEONFEEL96) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:422]:
```suggestion
"있습니다. 제어 열의 '모델 서빙' 버튼을 클릭하여 바로 모델 서비스를 시작할 수 있습니다. 아래에서는 'model_folder' 라는 폴더에 대해 "
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 5]
```diff
@@ -413,17 +422,18 @@ msgstr ""
...
 "모델 서비스를 시작해봅시다."
```
Comment (YEONFEEL96) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:423]:
```suggestion
"모델 서비스를 시작하는 방법에 대해 서술합니다. "
```

**Review by YEONFEEL96 — CHANGES_REQUESTED** [review]
I've made minor updates to the wording to align with the revisions in the current documentation.
However, there are certain aspects that cannot be modified via GithHub Suggestion at this point, which I will address in a subsequent PR.

**Review by YEONFEEL96 — COMMENTED** [review]
LGTM

**Comment by ironAiken2** [comment]
Can you change the status to approve? :)

**Review by YEONFEEL96 — APPROVED** [review]
Changed my stat to approved.

**Comment by ironAiken2** [comment]
### Merge activity

* **Nov 22, 12:36 AM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/lablup/backend.ai-docs-webui/89).

**Pull request merged**